src/ProductsXmlFeed.php

```diff
@@ -444,7 +451,7 @@ private static function get_property_g_additional_image_link( $product, $propert
 			return;
 		}
 
-		return '<' . $property . '><![CDATA[' . implode( ',', $images ) . ']]></' . $property . '>';
+		return '<' . $property . '><![CDATA[' . implode( ',', array_slice( $images, 0, self::ADDITIONAL_IMAGES_LIMIT ) ) . ']]></' . $property . '>';
```

Comment:
Minor: the `array_slice()` call should be applied to `$images` before the output string is returned. Alternatively, this would be more readable using `sprintf()` as a wrapper to generate this element.

Reply:
Thanks @JPry. Already pushed the changes for applying `array_slice()` and `implode()` before the output string.
About the `sprintf()`, after the changes the readability is much better, also the rest of returned stings follow the same pattern.